### PR TITLE
feat: render coordinates

### LINKED ISSUE #128: Show coordinates
Like the "inside the board" style on Lichess.

### PR DESCRIPTION
**Pull request by cymruu**
demo: 
```http://127.0.0.1:6175/image.gif?fen=4k3/6KP/8/8/8/8/7p/8&orientation=black&lastMove=a1a2```
image

```http://127.0.0.1:6175/image.gif?fen=4k3/6KP/8/8/8/8/7p/8&lastMove=h1h2```
image

```http://127.0.0.1:6175/example.gif```
https://github.com/cymruu/lila-gif/blob/cymruu/render-coordinates/example.gif?raw=true

closes #128

### CODE REVIEW

File: src/api.rs [hunk 1]
```diff
@@ -102,6 +102,58 @@ impl<'de> Deserialize<'de> for CheckSquare {
...
+                fmt.write_str("\"yes\", \"1\", \"no\" or bool")
+            }
+
+            fn visit_str<E>(self, name: &str) -> Result<Coordinates, E>
+            where
+                E: de::Error,
+            {
+                if name == "1" || name == "yes" || name == "true" {
+                    Ok(Coordinates::Yes)
+                } else {
```
Comment (niklasf) [inline src/api.rs:137]: Let's report an error if the parameter is not a known falsy name.

**Review by niklasf — CHANGES_REQUESTED** [review]
Thanks! This looks pretty good, already.

On Lichess, the coordinates are rendered in the opposite square color, e.g., light brown on dark brown and vice versa. Maybe that would be a bit more legible?

**Comment by cymruu** [comment]
@niklasf Thanks for reviewing. I have addressed your comments. Here are the results:
image

image

https://github.com/lichess-org/lila-gif/raw/7de92acac80dfc74377866bc2ebf5857a5124927/example.gif

**Comment by niklasf** [comment]
Looks good! Deployed to prod.

**Pull request merged**